charts/consul/templates/client-daemonset.yaml

```diff
@@ -124,9 +126,12 @@ spec:
         - name: config
           configMap:
             name: {{ template "consul.fullname" . }}-client-config
+        - name: consul-data
+          emptyDir:
+            medium: "Memory"
         {{- if .Values.global.tls.enabled }}
-        {{- if not .Values.global.secretsBackend.vault.enabled }}
         - name: consul-ca-cert
+          {{- if not .Values.global.secretsBackend.vault.enabled }}
```

Comment:
I think this should be above like it was originally so that this volume is not added when vault is enabled

Reply:
oh right!  thanks!

Reply:
Done.